src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -529,7 +529,7 @@ private void leaveTheRoom()
             chatRoom.leave();
         }
 
-        chatRoom.setListener(null);
+        chatRoom.removeListener(null);
```

Comment:
The argument here should be `chatRoomListener` not `null`.

Reply:
Good catch, I also annotated the interface.